packages/design-system-docs/src/pages/components/Pagination/_Pagination.docs.scss

```diff
@@ -0,0 +1,83 @@
+
+Pagination also comes in two styles: default, which is compact on mobile viewports and expands to reveal individual pages on larger viewports, and `compact`, which maintains the compact layout regardless of viewport size.
+
+@react-example Pagination.example.jsx
+
+@react-props Pagination.tsx
+
+Style guide: components.pagination.react
+*/
+
+/*
+---
+
+### When to use
+
+- Typically used for paginated search results.
+- Can be used for multi-page collections of related items, including articles related to a category or tag, content archives, and history or activity. Splitting a large collection of related items into individual pages can improve browsability and scannability.
+
+### When to consider alternatives
+
+- If you need to indicate progress in a series of steps that must be completed in succession, like an onboarding or checkout flow, this isn't the component for you.
+- If the length of the entire collection is less than 3-4 screen lengths long, consider showing all the items at once instead of paginating.
+
+### Usage
+
+- Users want to know the length of a paginated section, so show the size of the paginated set.
+- Highlight the current page the user is on in relation to the entire collection of pages.
+- Don't split the navigation items over multiple lines as this can make individual pages more difficult to understand and select.
+- Don't include out-of-sequence items directly adjacent to one another. Wherever there are missing pages, an ellipses should be used.
+- Avoid adding complexity by focusing on the essentials and avoid adding more items to Pagination just to fill the space.
+- Use touch targets that are big enough to select with any finger and have enough separation to avoid mistakes.
+- Do not use buttons for URL-based pagination. Because the URL is updated, the browser history stack updates and a link is the correct tag to use.
+- Consider page load, performance, and the user's scrolling preferences when determining how many items are displayed on each page.
+
+### Style customization
+
+The following Sass variables can be overridden to theme alerts:
+
+- `$pagination-nav-link-color` - Used to set color of next/previous and page number links.
+- `$pagination-nav-link-color-hover` - Used to set color of next/previous and page number links on hover.
+- `$pagination-nav-link-color-active` - Used to set color of next/previous and page number links when active.
+- `$pagination-nav-link-color-focus` - Used to set color of next/previous and page number links on focus.
+
+- `$pagination-current-page-color` - Used to set the current page element text color.
+- `$pagination-current-page-color-bg` - Used to set the current page element baclground color.
+- `$pagination-overflow-color` - Used to set the overflow element text color.
+- `$pagination-page-count-color` - Used to set the page count element text color.
+
+### Accessibility
```

Comment:
Is it worth adding a guideline in here about shifting focus to a `h1` or other element after the links are clicked? It's tangental to the component, but could be helpful.